lib/checkPullRequestReview.js

```diff
@@ -74,8 +79,28 @@ const handleChangesRequested = async (context, pullRequest, review) => {
       commentBody
     );
   }
+
+  if (allLabels.includes(LGTM_LABEL)) {
+    await context.github.issues.removeLabel(
+      context.repo({
+        issue_number: pullRequest.number,
+        name: [LGTM_LABEL]
+      })
+    );
+
+    await context.github.issues.createComment(
+      context.repo({
+        issue_number: pullRequest.number,
+        body: 'Hi, @' + pullRequest.assignee.login +
+        ', Removing LGTM Label since the changes are requested. Thanks!.',
+      }
+      )
+    );
```

Comment:
```suggestion
    await context.github.issues.createComment(
      context.repo({
        issue_number: pullRequest.number,
        body: 'Hi, @' + pullRequest.assignee.login +
        ', Removing LGTM Label since the changes are requested. Thanks!.',
      })
    );
```

Reply:
Making it that way sir.